client/src/components/ExploreGames/index.js

```diff
@@ -39,6 +49,7 @@ class ExploreGames extends React.Component {
     M.AutoInit()
   }
```

Comment:
so example 
<<<<< HEAD 
means that's what the master branch has on file.
====== is a divider of where the merge conflict starts (aka: the code you're trying to push)
and <<<<<< f15c0c757a75f208fb94e98a3a829fed477dee0a 
is where the end of the code conflict ends (the part of your code conflicting with the code in the master branch).